### PR TITLE
Add disclaimer if preview is not supported by user's framework

### PR DESCRIPTION
**Pull request by hypervillain**
If a user's project is not Next/Nuxt, don't display the "take screenshot" button, and prevent them to call "/api/screenshot". Also, give more information regarding Storybook. A link should be added to our docs once the page exists.

https://linear.app/prismic/issue/SM-369/aauser-if-i-use-a-framework-not-supported-by-the-preview-i-see-a

### CODE REVIEW

File: packages/slice-machine/tests/screenshot.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { validateEnv } from "../server/src/api/screenshots/screenshots";
```
Comment (Miniplop) [inline packages/slice-machine/tests/screenshot.test.js:1]: Nice 👍

File: packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx [hunk 2]
```diff
@@ -66,46 +116,21 @@ const SideBar: React.FunctionComponent<SideBarProps> = ({
...
       </Card>
       {!isPreviewAvailableForFramework ? (
-        <InformationBox>
-          <Text as="p" variant="xs" sx={{ mb: 2 }}>
-            Slice preview is not available for this framework
+        <>
+          <Button variant="disabled" sx={{ width: "100%", mt: 3 }}>
+            Open Slice Preview
+          </Button>
+          <Text as="p" sx={{ textAlign: "center", mt: 3, color: "#667587" }}>
```
Comment (Miniplop) [inline packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx]: we don't have style variable for this color ?

File: packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx [hunk 3]
```diff
@@ -78,14 +78,16 @@ const ImagePreview: React.FC<ImagePreviewProps> = ({
...
-                    onClick={onScreenshot}
-                  >
-                    Take screenshot
-                  </Button>
+                  {!preventScreenshot ? (
+                    <Button
+                      sx={{ mb: 3 }}
+                      variant="primary"
+                      disabled={preventScreenshot}
+                      onClick={onScreenshot}
```
Comment (Miniplop) [inline packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx:85]: maybe we can remove this line as the component wont be rendered with disabled = true

Comment (hypervillain) [reply packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx:85]: You mean the 'disabled' prop, right?

File: packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx (hunk 2 above)
Comment (hypervillain) [reply packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx]: textGray!

File: packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx (hunk 3 above)
Comment (Miniplop) [reply packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx:85]: yes :)

**Review by Miniplop — APPROVED** [review]

**Pull request merged**